assets/styles/_variables.scss

```diff
@@ -175,4 +175,13 @@ $aspect-ratios: (
   --kbin-footer-link-color: #fff;
   --kbin-footer-link-hover-color: var(--kbin-header-link-hover-color);
 
+  // details
+  --mbin-details-border: var(--kbin-section-border);
+  --mbin-details-separator-border: var(--kbin-meta-border);
+
+  --mbin-details-detail-color: var(--kbin-link-hover-color);
+  --mbin-details-spoiler-color: var(--kbin-danger-color);
+
+  --mbin-details-detail-label: "Details";
+  --mbin-details-spoiler-label: "Spoiler";
```

Comment:
might be wrong but guessing these don't get translated. since they only appear when a user doesn't specify a summary, doesn't seem worth holding up the change to me

Reply:
oops yeah, this entire details label dance is meant to provided a translated default label for a details block that doesn't have user label already, as I find leaving them empty to look a bit weird.

connected the translation using templating these label properties as inline styles, I wish there's better way to deliver these but this is the best I could think of at the moment.